src/style.css

```diff
@@ -2,0 +1,29 @@
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  margin: 0;
+}
+
+.stars__star {
```

Comment:
Be consistent with your margins. It's better to add only top or bottom margins, but not both, to avoid potential margin collapse issues.